packages/test/test-version-utils/src/compatConfig.ts

```diff
@@ -272,6 +272,7 @@ export async function mochaGlobalSetup() {
 		}
 	}
 	if (error) {
+		// eslint-disable-next-line @typescript-eslint/no-throw-literal -- rethrowing the originally caught value
 		throw error;
```

Comment:
eslint: rule is correct: we could potentially throw a literal here.  This is by-design as we're rethrowing the same value caught above.